README.md

```diff
@@ -1,55 +1,142 @@
+3. The signing server starts with default configuration in the `local-signer` container.
+4. Once the signing server is ready, the example runs a self-check using the Python client and verifies that a default image placed in `client_volume/signed-images` can be signed using a `local-client` container. You can then also see the signed test file in a directory created at the root of this repo, `client_volume/signed_images`.
+
+When the `make local` command finishes, it displays something like this:
+
+```shell
+(...Docker images build details...)
+
+--- Running containers.........
+
+docker compose up -d
+[+] Running 5/5
+ ✔ Network c2pa-python-example_default  Created                                                              0.0s
+ ✔ Container localstack-main            Started                                                              0.3s
+ ✔ Container local-setup                Exited                                                               0.8s
+ ✔ Container local-signer               Healthy                                                              6.4s
+ ✔ Container local-client               Started
+```
+
+### Re-run the Python client container
+
+To re-run the Python client, enter the following command from the root of this repository:
+
+```shell
+docker compose run --entrypoint "Python tests/client.py ./tests/A.jpg -o client_volume/signed-images" client
+```
+
+Replace `./tests/A.jpg` with the path to the image you want to sign.
+
+### Clean up the local setup
+
+After trying out the example, be sure to stop and remove the containers by running this command at the root of this repository:
+
+```shell
+make clean
+```
+
+Once the cleanup is done, you'll see messages similar to this:
+
+```shell
+
+--- Cleaning up.................
+
+docker compose down --volumes --remove-orphans
+[+] Running 6/6
+ ✔ Container local-client                 Removed                                                            0.0s 
+ ✔ Container local-signer                 Removed                                                           10.1s 
+ ✔ Container localstack-main              Removed                                                            1.1s 
+ ✔ Container local-setup                  Removed                                                            0.0s 
+ ✔ Volume c2pa-Python-example_local-data  Removed                                                            0.0s 
+ ✔ Network c2pa-Python-example_default    Removed
+```
+
+## Setting up to run the example locally
```

Comment:
Change to "Run the example locally" to match prior section and since this section covers not just setup but the whole process.

Reply:
Done